tests/groupTest.js

```diff
@@ -35,8 +35,8 @@ describe('Main: Currently testing group management,', function () {
           })
             .appendElement('data', '', (data) => {
               data.appendElement('groups', '', (groups) => {
-                groups.appendElement('element', '', 'admin')
-                groups.appendElement('element', '', config.testGroup)
+                groups.appendElement('element', '', MatchersV3.string(config.adminUsername))
+                groups.appendElement('element', '', MatchersV3.string(config.testGroup))
```

Comment:
I don't think we need matchers here, the mock server and the provider should respond with the exact values